src/components/FileMenu/FileMenu.js

```diff
@@ -169,6 +157,17 @@ export const FileMenu = ({
                     {i18n.t('File')}
                 </button>
             </div>
+            <OpenFileDialog
+                open={openDialog === 'open'}
```

Comment:
can the `'open'` string be made in to a const instead?

Reply:
also, can we rename `openDialog` to something like `currentDialog`? It's quite confusing to have a prop that refers to "which dialog is open" and a component with a similar name that refers to "open up a file"

Reply:
and then you have `onOpen` and `menuIsOpen`... everything is "open" here 😄  would be great to give things more specific names 🧠 🌀

Reply:
Renamed `currentDialog` but left the other 2.
`onOpen` for consistency with the other callback props which follow the button names, and for backwards compatibility so there is no need to change the apps using this component.